reference/camera.md

```diff
@@ -29,8 +29,7 @@ eye which is spherically distorted.
 
 ### Field Summary
 
-- `fieldOfView`: horizontal field of view angle of the camera. The value ranges
-from *0* to π radians. Since camera pixels are squares, the vertical field of
+- `fieldOfView`: horizontal field of view angle of the camera. The value is limited to the range *0* to π radians if the `spherical` field is set to FALSE, otherwise there is no upper limit. Since camera pixels are squares, the vertical field of
```

Comment:
0 and pi should be either both highlighted or none highlighted. Currently only 0 is highlighted.